src/cryptography/hazmat/bindings/openssl/binding.py

```diff
@@ -38,8 +43,12 @@ def _openssl_assert(lib, ok):
     if not ok:
         errors = _consume_errors(lib)
         raise InternalError(
-            "Unknown OpenSSL error. Please file an issue at https://github.com"
-            "/pyca/cryptography/issues with information on how to reproduce "
+            "Unknown OpenSSL error. This error is commonly encountered when "
+            "another library is not cleaning up the OpenSSL error stack. If "
+            "you are using cryptography with another Python library that "
+            "uses OpenSSL try disabling it before reporting a bug. Otherwise "
+            "iplease file an issue at https://github.com/pyca/cryptography/"
```

Comment:
teehee

On Mon, Mar 7, 2016 at 8:51 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/bindings/openssl/binding.py
> <https://github.com/pyca/cryptography/pull/2774#discussion_r55305598>:
>
> > @@ -38,8 +43,12 @@ def _openssl_assert(lib, ok):
> >      if not ok:
> >          errors = _consume_errors(lib)
> >          raise InternalError(
> > -            "Unknown OpenSSL error. Please file an issue at https://github.com"
> > -            "/pyca/cryptography/issues with information on how to reproduce "
> > +            "Unknown OpenSSL error. This error is commonly encountered when "
> > +            "another library is not cleaning up the OpenSSL error stack. If "
> > +            "you are using cryptography with another Python library that "
> > +            "uses OpenSSL try disabling it before reporting a bug. Otherwise "
> > +            "iplease file an issue at https://github.com/pyca/cryptography/"
>
> Definitely not an artifact of sucking at vim, no sir.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2774/files#r55305598>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084